microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -950,7 +934,7 @@ This data is not sent during InPrivate browsing or Guest mode.
 <!-- ====================================================================== -->
 ## Tracking prevention
 
-Microsoft Edge is designed to detect and block known trackers.  Users may choose from three levels of tracking prevention: Basic, Balanced, and Strict.  To protect user privacy, Balanced is selected by default.  Microsoft Edge detects trackers before any are loaded on the page by using an open-source list of known trackers.  The list is downloaded to the device periodically as the list is updated.  The number of trackers blocked and names of those trackers are stored locally on the device for statistical purposes.  To clear the data, go to `edge://settings/privacy/blockedTrackers`.  The detection and blocking of trackers occurs locally on the device.  To disable tracking prevention, go to `edge://settings/privacy`.  For more information about Tracking prevention, see [Learn about tracking prevention in Microsoft Edge](https://support.microsoft.com/help/4533959).
+Microsoft Edge is designed to detect and block known trackers.  Users may choose from three levels of tracking prevention: Basic, Balanced, and Strict.  To protect user privacy, Balanced is selected by default.  Microsoft Edge detects trackers before any are loaded on the page by using an open-source list of known trackers.  The list is downloaded to the device periodically as the list is updated.  The number of trackers blocked and names of those trackers are stored locally on the device for statistical purposes.  To clear the data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security** > **Blocked trackers**.  The detection and blocking of trackers occurs locally on the device.  To disable tracking prevention, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security**.  For more information about Tracking prevention, see [Learn about tracking prevention in Microsoft Edge](https://support.microsoft.com/help/4533959).
```

Comment:
Privacy and security.

Reply:
probably Delete this comment, it's in wrong place; in latest Files Changed view, there's no "todo" re: edge://settings/privacy here.  @PatrizioBytes

Reply:
I'm adding `and then turn off the setting **Tracking prevention**` here, to match the other instances. But, yes, we can resolve this comment.